contents/handbook/people/team-structure/app-west.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Team App West
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- @EDsCODE (Team lead)
+- @clarkus (Designer)
+- @rcmarron (Engineer)
+- @liyiy (Engineer)
+- @alexkim205 (Engineer)
+- @neilkakkar (Engineer)
+
+## Mission
+
+Makers everywhere get better at building products because of PostHog
+
+## Q3 2022 Goals
+* **Proposed Objective:**
+    * Increase the % of customers successfully using multiple core tools
```

Comment:
What qualifies as a core tool here? 

- Dashboards and insights
- Recordings
- Feature flags
- Experiments
- Do apps qualify?
- Toolbar?

Reply:
Good point, (if we're think the proposed key results make sense) we should specify the tools in the objective, e.g. More customers are using insights with recordings and feature flags

Reply:
```suggestion
    * More customers successfully use recordings, experiments and feature flags with insights
```